### PR TITLE
feat: Docs for Sentry.span

### PR DESCRIPTION
**Pull request by philipphofmann**
Add missing docs for this feature and clarify which span to return depending on global mode.

### CODE REVIEW

File: src/docs/sdk/performance/index.mdx [hunk 1]
```diff
@@ -219,11 +219,21 @@ Which in reality is useful for proxies to set it to `0` and opt out of tracing.
...
+With `Sentry.span` users can attach spans to an already ongoing transaction.
+This property returns a `SpanProtocol` if a running transaction is bound to
+the scope; otherwise, it returns nil. Although we recommend users keep track
+of their own transactions, the SDKs should offer a way to expose auto-generated
+transactions. SDKs shall bind auto-generated transactions to the scope, making
+them accessible with `Sentry.span`.
+If the SDK has global mode enabled, which specifies whether to use global scope
+management mode and should be `true` for client applications and `false` for server
+applications, `Sentry.span` shall return the root span of the transaction. If the
+user disables global mode, `Sentry.span` shall return the latest unfinished span.
```
Comment (marandaneto) [inline src/docs/sdk/performance/index.mdx:235]:
```suggestion
user disables global mode, `Sentry.span` shall return the latest active (unfinished) span.
```

**Review by marandaneto — APPROVED** [review]

File: src/docs/sdk/performance/index.mdx [hunk 2]
```diff
@@ -219,11 +219,21 @@ Which in reality is useful for proxies to set it to `0` and opt out of tracing.
 ## Static API Changes
 
 The `Sentry.startTransaction` function should take two arguments - the `transactionContext` passed to the `Transaction` constructor and an optional `customSamplingContext` object containing data to be passed to `tracesSampler` (if defined).
-
 It creates a `Transaction` bound to the current hub and returns the instance.
 Users interact with the instance for creating child spans and, thus, have to
 keep track of it themselves.
 
+With `Sentry.span` users can attach spans to an already ongoing transaction.
```
Comment (bruno-garcia) [inline src/docs/sdk/performance/index.mdx:226]: In .NET it's `SentrySdk.GetSpan` which is quite different, is there a way to make the name included?

E.g: refer to "get span from static Sentry API" and at the top explain: for example `Sentry.span`, `SentrySdk.GetSpan`?

File: src/docs/sdk/performance/index.mdx [hunk 3]
```diff
@@ -219,11 +219,21 @@ Which in reality is useful for proxies to set it to `0` and opt out of tracing.
...
+This property returns a `SpanProtocol` if a running transaction is bound to
```
Comment (mattjohnsonpint) [inline src/docs/sdk/performance/index.mdx:227]: I believe `SpanProtocol` is specific to Cocoa.

Also, there's a section just below here in the same file titled `Scope Changes` that describes `setSpan`.  I think this section belongs there, and can use similar generic terminology as that paragraph.

File: src/docs/sdk/performance/index.mdx [hunk 4]
```diff
@@ -219,11 +219,21 @@ Which in reality is useful for proxies to set it to `0` and opt out of tracing.
...
+the scope; otherwise, it returns nil. Although we recommend users keep track
+of their own transactions, the SDKs should offer a way to expose auto-generated
+transactions. SDKs shall bind auto-generated transactions to the scope, making
+them accessible with `Sentry.span`.
+If the SDK has global mode enabled, which specifies whether to use global scope
```
[the remark this answers is not in the document]
Comment (philipphofmann) [reply src/docs/sdk/performance/index.mdx:232]: I'm unsure if we want to add this to the unified API spec.

**Comment by philipphofmann** [comment]
I merged this now, as this PR has been hanging here for a while now. I can make changes with a future PR if you like @bruno-garcia and @mattjohnsonpint.

**Pull request merged**